joint_trajectory_controller/src/joint_trajectory_controller.cpp

```diff
@@ -942,7 +942,7 @@ controller_interface::CallbackReturn JointTrajectoryController::on_configure(
   interpolation_method_ = interpolation_methods::from_string(interpolation_string);
   RCLCPP_INFO(
     logger, "Using '%s' interpolation method.",
-    interpolation_methods::InterpolationMethodMap.at(interpolation_method_).c_str());
+    interpolation_methods::ReverseInterpolationMethodMap.at(interpolation_method_).c_str());
```

Comment:
```suggestion
    interpolation_string.c_str());
```
Can't we use this directly?